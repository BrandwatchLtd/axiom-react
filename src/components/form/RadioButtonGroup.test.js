import React from 'react';
import renderer from 'react-test-renderer';
import { RadioButton, RadioButtonGroup } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <RadioButtonGroup { ...props }>
      <RadioButton>Lorem ipsum</RadioButton>
    </RadioButtonGroup>
  );
}

describe('RadioButton', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
