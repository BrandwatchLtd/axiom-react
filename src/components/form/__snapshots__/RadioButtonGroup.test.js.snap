// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton renders with defaultProps 1`] = `
<div
  className="ax-chedio-buttox-group"
>
  <label
    className="ax-chedio-buttox__container ax-space--x2"
    onClick={undefined}
  >
    <input
      className="ax-chedio-buttox ax-radio"
      disabled={undefined}
      name={undefined}
      type="radio"
    />
    <span
      className="ax-chedio-buttox__indicator ax-radio__indicator"
    />
    <span
      className="ax-chedio-buttox__label"
    >
      Lorem ipsum
    </span>
  </label>
</div>
`;
